draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1566,8 +1566,9 @@ Steps:
 8. return sign
 ~~~
 
-Note that any sgn0 function for extension fields must iterate over
-the entire vector representation of the input element.
+The definition of the sgn0 function for extension fields relies on
```

Comment:
This change makes the rest of the paragraph sort of irrelevant. If we're going to take this, then we should drop lines 1572-1576, too.

Reply:
Change was made and merged with previous line.